api/src/main/resources/ConfigurationEntry.hbm.xml

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE hibernate-mapping PUBLIC "-//Hibernate/Hibernate Mapping DTD 3.0//EN" "http://hibernate.sourceforge.net/hibernate-mapping-3.0.dtd">
+<hibernate-mapping>
+	<class name="org.openmrs.module.patientmatching.ConfigurationEntry" table="patientmatching_configurationEntries">

```

Comment:
@bmamlin this has a slight problem because the default table name is ** patientmatching_configurationEntries**, which is provided by the OpenMRS database. So I guess I cannot change it can I?